meta_model_building/get_opt_params.py

```diff
@@ -0,0 +1,40 @@
+import os
+import pickle
+from tpot import TPOTRegressor
+from sklearn.datasets import load_boston
+from sklearn.model_selection import train_test_split
+import time
+
+run_param = {
+	'random_state_train': 42,
+	'random_state_tpot': 42,
+	'export_python_code_filename': 'tpot_boston_pipeline.py',
+	'pickle_file_name': 'boston_pipeline_evaluated_models.pickle',
+	'train_size': 0.75,
+	'test_size': 0.25,
+	'generations': 5,
+	'population_size': 20,
+	'verbosity': 2,
+	'time': time.time()
+}
+
+housing = load_boston()
+X_train, X_test, y_train, y_test = train_test_split(housing.data, housing.target,
```

Comment:
In general I'm not a fan of mixing upper and lower case like this.

Reply:
Fixed in second commit to dev_get-boston-data_20180923